Sources/Request/Request/Request.swift

```diff
@@ -43,7 +43,7 @@ public typealias Request = AnyRequest<Data>
 public struct AnyRequest<ResponseType> where ResponseType: Decodable {
     public let combineIdentifier = CombineIdentifier()
 
-    private var params: CombinedParams
+    private var params: [RequestParam]
```

Comment:
If you remove unzip, this should be switched back to:
```suggestion
    private var params: RequestParam
```